frame/staking/src/pallet/impls.rs

```diff
@@ -1001,7 +1012,7 @@ impl<T: Config> ElectionDataProvider for Pallet<T> {
 			);
 			Self::do_add_nominator(
 				&v,
-				Nominations { targets: t, submitted_in: 0, suppressed: false },
+				Nominations { targets: t.try_into().unwrap(), submitted_in: 0, suppressed: false },
```

Comment:
```suggestion
				Nominations { targets: t.try_into().expect("... .qed"), submitted_in: 0, suppressed: false },
```

Reply:
I think it would be better to add a justification here

Reply:
actually we don't. This function is designed for being called by the benchmarking setup code, and we can't have a guarantee that they call it with proper parameters, thus you can't have a proof style `.expect`.

Since this is not production code, `.unwrap` is perfectly fine IMO.

Reply:
oh my bad, I think I got confused with something else and didn't realize this was benchmarking code. Totally agree with you